packages/stores/types.ts

```diff
@@ -448,6 +448,10 @@ type TTradersHubStore = {
     selected_region: TRegionAvailability;
     getExistingAccounts: (platform: string, market_type: string) => AvailableAccount[];
     available_dxtrade_accounts: AvailableAccount[];
+    active_modal_tab_index: number;
+    setWalletModalActiveTabIndex: (index: number) => void;
+    active_modal_wallet_id?: string;
+    setWalletModalActiveWalletID: (wallet_id?: string) => void;
```

Comment:
@george-usynin-deriv We should plan to move `WalletModal` out of `ModalManager`, The number of variables in stores are growing just because we can't pass prop to `WalletModal` :(